components/org.wso2.carbon.micro.integrator.extensions/org.wso2.carbon.micro.integrator.management.apis/src/main/java/org/wso2/carbon/micro/integrator/management/apis/ApiResource.java

```diff
@@ -210,52 +159,37 @@ private OMElement convertApiToOMElement(API api, MessageContext messageContext)
 
         String version = api.getVersion().equals("") ? "N/A" : api.getVersion();
 
-        versionElement.setText(version);
-        rootElement.addChild(versionElement);
-
-        String statisticState = api.getAspectConfiguration().isStatisticsEnable() ? "enabled" : "disabled";
+        apiObject.put(Constants.VERSION, version);
 
-        statsElement.setText(statisticState);
-        rootElement.addChild(statsElement);
+        String statisticState = api.getAspectConfiguration().isStatisticsEnable() ? Constants.ENABLED : Constants.DISABLED;
+        apiObject.put(Constants.STATS, statisticState);
 
-        String tracingState = api.getAspectConfiguration().isTracingEnabled() ? "enabled" : "disabled";
+        String tracingState = api.getAspectConfiguration().isTracingEnabled() ? Constants.ENABLED : Constants.DISABLED;
+        apiObject.put(Constants.TRACING, tracingState);
 
-        tracingElement.setText(tracingState);
-        rootElement.addChild(tracingElement);
-
-        OMElement resourcesElement = AXIOMUtil.stringToOM(RESOURCES_ELEMENT);
-        rootElement.addChild(resourcesElement);
+        JSONArray resourceListObject = new JSONArray();
+        apiObject.put("resources", resourceListObject);
 
         Resource[] resources = api.getResources();
 
         for (Resource resource : resources) {
 
-            OMElement resourceElement = AXIOMUtil.stringToOM(RESOURCE_ELEMENT);
-            resourcesElement.addChild(resourceElement);
-
-            OMElement methodElement = AXIOMUtil.stringToOM(METHOD_ELEMENT);
-            resourceElement.addChild(methodElement);
-
-            OMElement urlElement = AXIOMUtil.stringToOM(URL_ELEMENT);
-            resourceElement.addChild(urlElement);
+            JSONObject resourceObject = new JSONObject();
 
             String[] methods = resource.getMethods();
 
-            for (String method : methods) {
-                OMElement itemElement = AXIOMUtil.stringToOM(LIST_ITEM);
-                itemElement.setText(method);
-                methodElement.addChild(itemElement);
-            }
+            resourceObject.put("methods", methods);
 
             DispatcherHelper dispatcherHelper = resource.getDispatcherHelper();
             if (dispatcherHelper instanceof URITemplateHelper) {
-                urlElement.setText(dispatcherHelper.getString());
+                resourceObject.put(Constants.URL, dispatcherHelper.getString());
 
             } else if (dispatcherHelper instanceof URLMappingHelper) {
-                urlElement.setText(dispatcherHelper.getString());
+                resourceObject.put(Constants.URL, dispatcherHelper.getString());
```

Comment:
"All branches in a conditional structure should not have exactly the same implementation" - SonarLint error

Reply:
Similar warning in line 232.